pkg/network/ovn_kubernetes.go

```diff
@@ -114,14 +114,6 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 
 	if c.IPsecConfig != nil {
```

Comment:
data.Data["EnableIPsec"] = c.IPSecConfig != nil

Reply:
this is a nit. you don't have to fix it if all the other jobs are passing right now. it's more important than fixing the nit and having jobs fail again because - CI gods

Reply:
I think it should pass so I'll continue playing the retest lottery